ccd-yarn-audit-known-issues

```diff
@@ -0,0 +1,3 @@
+https://npmjs.com/advisories/1179
+https://npmjs.com/advisories/1500
+https://npmjs.com/advisories/1544
```

Comment:
I think we should fix these if possible. The first two I'm pretty sure have been fixed in the Node services; the third might be fixable in a similar manner.

Reply:
Have fixed the vulnerabilities